lib/Munin/Node/Configure/Plugin.pm

```diff
@@ -75,6 +75,16 @@ sub suggestion_string
         # Report why it's not being used
         $msg = " [$self->{defaultreason}]";
     }
+    elsif (! $self->{capabilities}->{autoconf}) {
+        # No autoconf support
+        $msg = " [[[ plugin has no autoconf support ]]]";
+    }
+    elsif ( scalar @{$self->{errors}} != 0 ) {
+        $msg = " [[[ plugin has errors, see below ]]]";
```

Comment:
Do you have something specific in mind with the three square brackets? The previous message above uses just one - probably you can stick to this pattern here, too?

Reply:
I wanted to make it distinguishable if the plugins supplied a message or munin itself has something to say; but I can change it